src/blockstore/commitor.ts

```diff
@@ -163,7 +163,7 @@ async function prepareCarFiles(
 ): Promise<{ cid: AnyLink; bytes: Uint8Array }[]> {
   // const theKey = isPublic ? undefined : await this._getKey();
   const carFiles: { cid: AnyLink; bytes: Uint8Array }[] = [];
-  threshold = threshold || 1000 * 1000;
+  threshold = threshold || 128000;
```

Comment:
this low default is for PK testing -- @mabels @mschoch can you help me configure this or the `fragSize` here? https://github.com/fireproof-storage/connect/blob/46f6fcf96cf4a10014affb5c3f4794f768abd017/src/partykit/index.ts#L46